nexus/api_test.go

```diff
@@ -99,7 +100,7 @@ func TestAddLinksToHeader(t *testing.T) {
 				Type: "url",
 			}},
 			output: http.Header{
-				headerLink: []string{
+				textproto.CanonicalMIMEHeaderKey(headerLink): []string{
```

Comment:
Why use this library over `http.CanonicalHeaderKey`?

Reply:
I missed this, they’re essentially the same function but it’s cleaner to use the function exported in http. Thanks!